.github/workflows/test.yml

```diff
@@ -55,6 +55,10 @@ jobs:
         ref: ${{ github.event.pull_request.head.sha }}
         submodules: recursive
 
+    - name: apt update
+      run: |
+        sudo apt update
```

Comment:
[note to self] this is here because the next step involves `make lint-setup` which uses `sudo apt-get`